compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDistributeCoresAndObjectFifos.cpp

```diff
@@ -238,6 +263,7 @@ class AMDAIEUnrollLocalLoops : public OpRewritePattern<scf::ForOp> {
   /// scf.for. This assumption could be removed in the future.
   template <typename Iterator>
   LogicalResult hoistDmaOps(PatternRewriter &rewriter, scf::ForOp forOp) const {
+    if (forOp->getParentOfType<AMDAIE::CoreOp>()) return failure();
```

Comment:
Why is this change needed?

Reply:
The hoisting of DmaOp concept shouldn't be needed for a vectorized loop nest. We can bail out quite early for such cases.

I've now added that as a comment.